### ISSUE TITLE
After login to channel which create non admins, I see only a white screen

### ISSUE THREAD

**Issue by realyenman** [mattermost/mattermost-mobile#2482]
#### Summary
After login to channel which create non admins - I see only a white screen

#### Environment Information

- Device Name: Samsung Galaxy S9+
- OS Version: Android 8.0.0
- Mattermost App Version: 1.15.1
- Mattermost Server Version: 5.6.1

#### Steps to reproduce
Create a public channel through the desktop application and try to login it on the mobile application.

#### Expected behavior
After login to channel which create non admins - I see only a white screen. 

#### Observed behavior
![ezgif-4-a2a67628b43b](https://user-images.githubusercontent.com/46228046/50533449-bee1f980-0b3a-11e9-9597-7d209dba77f1.gif)

**Comment by enahum** [comment]
Is that channel empty or it contains posts? Cause it could be that is failing to render a specific post. If that is the case it would help to narrow down which post is it so a fix can be applied.

**Comment by realyenman** [comment]
> Is that channel empty or it contains posts? Cause it could be that is failing to render a specific post. If that is the case it would help to narrow down which post is it so a fix can be applied.

There are no posts in the channel.

**Comment by jasonblais** [comment]
@realyenman Would you be open to changing your language to English (via Account Settings > Display > Language), then trying again if this reproduces for you?

I tried with Russian translations myself, but wasn't able to reproduce it yet.

**Comment by realyenman** [comment]
If i change language to English on the desktop client - the channel starts to open normaly. If i change the language back to Russian - it continues to work like a charm. But this is not a solution. I can not understand the patterns. In reviews on Google Play, there are many of the same as my reviews from users from Russia.

**Comment by jasonblais** [comment]
Now that you switched back to Russian, does creating a public channel through the desktop application and logging in the mobile app reproduce the issue, or it works great?

I'm not sure what might have caused it..did it happen to many users in your team?

**Comment by realyenman** [comment]
This is guaranteed with all users in the team. Mobile phones on Android and iOS. The channel that was the problem - it works fine. But if I create a new channel, users will enter it, after some time it will stop working on the mobile device and the user will need to switch the language in the settings again.

**Comment by lindalumitchell** [comment]
I'm able to reproduce this on 5.6 and 5.8 servers, with RN builds from latest back to v1.15.0. Does not reproduce for me on RN version v1.14. Reproduces for me as described: Only fails when set to Russian, works again when set to English and back, then fails again on Russian. Did not reproduce for me on a couple other languages I tried. I added details to the ticket: https://mattermost.atlassian.net/browse/MM-13974
